computation_graph/run.py

```diff
@@ -418,9 +447,46 @@ def _construct_computation_result(
         ),
         gamla.star(
             lambda result, state: base_types.ComputationResult(
-                result=result, state=state,
+                result=result,
+                state=state,
+            ),
+        ),
+    )
+
+
+def debug_trace(edges, result_dependencies):
+    if config.DEBUG_SAVE_COMPUTATION_TRACE:
+        from computation_graph import visualization
```

Comment:
I think we can import this at the top. Any reason to import conditionally ?

Reply:
to avoid depending on pygraphviz in production.